### PR TITLE
docs: troubleshooting rootful connection

### LINKED ISSUE #3045: Update the troubleshooting guide
- [ ] Update outdated info https://github.com/containers/podman-desktop/pull/3088 https://github.com/containers/podman-desktop/pull/3089
- [ ] Should take the things we suggest to issues into account when talking with users and helping debug in the troubleshooting guide
- [x] Add troubleshooting info for: https://github.com/containers/podman/issues/19554
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/2806
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/1633
- [ ] Add troubleshooting info for: https://github.com/containers/podman-desktop/issues/2794
- [x] Should update the troubleshooting guide with the new troubleshooting page feature https://github.com/containers/podman-desktop/pull/3083

### LINKED ISSUE #3617: podman desktop Images pandel  unloading local images.
### Bug description

podman desktop Images pandel  unloading local images.

display "No images"

image

Use the cmd.exe podman command. print "docker.io/jenkins/jenkins"
image

image

### Operating system

windows 10

### Installation Method

Installer from website/GitHub releases

### Version

1.3.1

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

docs: troubleshooting rootful connection

### What issues does this PR fix or reference?
references #3617

### CODE REVIEW

File: website/docs/troubleshooting/troubleshooting-podman.md [hunk 1]
```diff
@@ -128,6 +128,40 @@ Podman Desktop might display "No Containers" as shown below, even if there are a
    $ podman machine init
    ```
 
+## Podman Desktop is failing to display the images or containers from a rootlful Podman machine
```
Comment (benoitf) [inline website/docs/troubleshooting/troubleshooting-podman.md]:
```suggestion
## Podman Desktop is failing to display the images or containers from a rootful Podman machine
```

looks like there is a typo (don't know how it's not reported as a warning in checks)

File: website/docs/troubleshooting/troubleshooting-podman.md [hunk 2]
```diff
@@ -128,6 +128,40 @@ Podman Desktop might display "No Containers" as shown below, even if there are a
...
+   ```shell-session
+   $ podman machine set --rootful
+   ```
+
+1. Start the Podman machine:
+
+   ```shell-session
+   $ podman machine start
+   ```
+
```
Comment (benoitf) [inline website/docs/troubleshooting/troubleshooting-podman.md:200]: probably a step to check again with `podman system connection ls`

and if still not pointing the right connection, could use `podman system connection default podman-machine-default-root` (or the name of the machine)

Comment (themr0c) [reply website/docs/troubleshooting/troubleshooting-podman.md:200]: done

File: website/docs/troubleshooting/troubleshooting-podman.md [hunk 3]
```diff
@@ -128,6 +128,84 @@ Podman Desktop might display "No Containers" as shown below, even if there are a
...
+1. Verify that the Podman default connection is the rootful connection to your Podman machine:
+
+   ```shell-session
+   $ podman system connection ls
+   ```
+
+   The default connection has `true` at the end of the line.
+
+   The rootful connection has a `-root` name suffix, and a `ssh://root@`URI prefix.
```
Comment (benoitf) [inline website/docs/troubleshooting/troubleshooting-podman.md]:
```suggestion
   The rootful connection has a `-root` name suffix, and a `ssh://root@` URI prefix.
```

**Review by benoitf — APPROVED** [review]
just a minor typo

**Pull request merged**